addons/popochiu/engine/objects/gui/components/dialog_text/dialog_text.gd

```diff
@@ -73,14 +73,18 @@ func play_text(props: Dictionary) -> void:
 	
 	if PopochiuConfig.should_talk_gibberish():
 		msg = D.create_gibberish(msg)
-	
+
 	# Call the virtual method that modifies the size of the RichTextLabel in case the dialog style
 	# requires it.
 	await _modify_size(msg, props.position)
 	
+	rich_text_label.push_color(props.color)
+	
 	# Assign the text and align mode
-	msg = "[color=%s]%s[/color]" % [props.color.to_html(), msg]
-	_append_text(msg, props)
+	if PopochiuConfig.should_talk_gibberish():
+		_append_text(D.create_gibberish(msg), props)
+	else:
+		_append_text(msg, props)
```

Comment:
This should be just this since the gibberish text replacement was already done in line 75.

```suggestion
	_append_text(msg, props)
```